boards/shields/seeed_xiao_round_display/doc/index.rst

```diff
@@ -0,0 +1,40 @@
+.. _seeed_xiao_round_display:
+
+Seeed Studio XIAO Round Display
+#################################
+
+Overview
+********
+
+Seeed Studio Round Display for XIAO is an expansion board compatible with all
+XIAO development boards. It features a fully covered touch screen on one side,
+designed as a 39mm disc. It contains onboard RTC, charge chip, TF card slot
+within its compact size, perfect for interactive displays in smart home,
+wearables and more.
+
+More information can be found on `the Getting Started page`_
+
+.. figure:: img/seeed_xiao_round_display.webp
+     :align: center
+     :alt: Seeed Studio XIAO Round Display
```

Comment:
Image is taking a lot of space on the page :) (see https://builds.zephyrproject.io/zephyr/pr/71239/docs/boards/shields/seeed_xiao_round_display/doc/index.html) 

```suggestion
     :width: 450 px
     :alt: Seeed Studio XIAO Round Display
```

alternatively, resize the source image so that it's "natively" smaller

Reply:
Image resized ;)